services/datamanager/builtin/builtin_capture_test.go

```diff
@@ -148,6 +145,9 @@ func TestDataCaptureEnabled(t *testing.T) {
 			c.CaptureDir = initCaptureDir
 			c.CaptureDisabled = tc.initialServiceDisableStatus
 			c.ScheduledSyncDisabled = true
+			// MaximumCaptureFileSizeBytes is set to 1 so that each reading becomes its own capture file
+			// and we can confidently read the capture file without it's contents being modified by the collector
+			c.MaximumCaptureFileSizeBytes = 1
```

Comment:
Previously our tests that were asserting on the contents of capture files could read from .prog files when they were still being written to. I changed all tests that assert on the contents of capture files to have a max caputre file size of 1 byte which will ensure that only a single sensor reading is written to a .prog file after which it will be turned into a .capture file and the collector won't write to it again.